src/ch01-00-types.md

```diff
@@ -0,0 +1,37 @@
+# Types
+
+Mun is a statically typed languages which helps to detect type-related errors in
+a program at compile-time. A type error is an invalid operation on a given type
+such as a division by a string, trying to access a field that doesn't exist or
+calling a function with the wrong number of arguments.
+
+Some languages require a programmer to explicitly annotate syntactic constructs
+with type information:
+
+```c++
+int foo = 3+4;
+```
+
+Often the types of variables can be inferred by their usage just as in the
+example above. Mun uses type inferencing to determine the type of a variable at
+compile time. However, it does force you to explicitly annotate variables in a
+few locations to ensure a contract between interdependent code.
+
+```mun
+function bar(a:int) {
+    let foo = 3+a
+}
+```
+
+Here, the parameter `a` must be annotated because it solidifies the signature of
+the function. However, the type of `foo` can be inferred through it's usage.
+
+## Numeric types
+
+Mun knows two basic numeric types: `float` and `int`. A `float` is a
+double-precision IEEE 64-bit floating point number and an `int` represents a
+64-bit integral number.
+
+In Mun an `int` can be explicitly casted to `float` but the reverse is not true:
```

Comment:
```suggestion
In Mun an `int` can be explicitly cast to `float`, but the reverse is not true.
```

Reply:
Do you mean `implicitly cast`? Otherwise the next line should be changed to read `and can therefore not be cast explicitly`.